### PR TITLE
Optional HTTPS settings for the taxes fetch and edited error handling

### PR DESCRIPTION
**Pull request by NyanKiyoshi**
This extends https://github.com/mirumee/django-prices-vatlayer/pull/7.

- Added optional HTTPS settings for the taxes fetch + error logging;
- Added logging for errors allowing users to know what's wrong
and remove the assumption that the issue is the ACCESS KEY.

Draft: this PR still needs to get the edits from `django-prices-vatlayer`.

### Pull Request Checklist

1. [x] Privileged views and APIs are guarded by proper permission checks.
1. [x] All visible strings are translated with proper context.
1. [x] All data-formatting is locale-aware (dates, numbers, and so on).
1. [x] Database queries are optimized and the number of queries is constant.
1. [x] The changes are tested.
1. [x] The code is documented (docstrings, project documentation).

### CODE REVIEW

File: saleor/settings.py [hunk 1]
```diff
@@ -320,7 +320,9 @@ def get_bool_from_env(name, default_value):
 
 # VAT configuration
 # Enabling vat requires valid vatlayer access key.
+# If you are subscribed to a paid vatlayer plan, you can enable HTTPS.
 VATLAYER_ACCESS_KEY = os.environ.get('VATLAYER_ACCESS_KEY')
+VATLAYER_USE_HTTPS = get_bool_from_env('VATLAYER_USE_HTTPS', False)
```
Comment (Pacu2) [inline saleor/settings.py]: Can we use `ENABLE_SSL` variable here?

Comment (NyanKiyoshi) [reply saleor/settings.py]: The settings `ENABLE_SSL` to set `VATLAYER_USE_HTTPS`? That would not work as the issue is that most of the people that will use the taxes feature won't have a paid vatlayer plan (thus no HTTPS available for API requests).

Comment (Pacu2) [reply saleor/settings.py]: I didnt know that paid plan is required for this, Thanks for the clarification

**Comment by NyanKiyoshi** [comment]
This PR is ready for review as django-prices-vatlayer was updated.

**Pull request merged**